src/content/docs/data-acquisition/web-scraping.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Web Scraping
```

Comment:
From the conclusion, you seem to be focussing on the use of AI in web scrapping. Can you update the title to reflect the content of your article?

Reply:
the title is now updated to be more descriptive